training_data_collector/chessdotcom/labelgame.py

```diff
@@ -0,0 +1,47 @@
+from selenium.webdriver.common.by import By
+import cv2
+import os
+import shutil
+
+from chessdotcom import actions
+from darknetconvert import convert_co_ords_to_darknet
+
+def label_game(driver, game_id):
+    gameName = 'images/game1/'
+    moveCount = actions.get_move_count(driver)
+
+    try:
+        shutil.rmtree('images/game1')
+    except:
+        pass
+    os.mkdir(gameName)
+    # for x in range(1, moveCount + 1):
+    #     labelBoard(driver, x, gameName, game_id)
+    #     actions.nextMove(driver)  
+    # labelBoard(driver, x + 1, gameName, game_id)
```

Comment:
What's this commented out?

Reply:
Thats the game loop. Needs to stay. It's still a work in progress.